src/dumpulator/dumpulator.py

```diff
@@ -821,24 +680,17 @@ def _hook_code(uc: Uc, address, size, dp: Dumpulator):
     except StopIteration:
         instr = None  # Unsupported instruction
     except UcError:
-        instr = None # Likely invalid memory
+        instr = None  # Likely invalid memory
         code = []
-    address_name = dp.exports.get(address, "")
 
-    module = ""
-    if dp.last_module and dp.last_module.baseaddress <= address < dp.last_module.baseaddress + dp.last_module.size:
-        # same module again
-        pass
-    else:
-        # new module
-        dp.last_module = dp.find_module_by_addr(address)
-        if dp.last_module:
-            module = dp.last_module.name.split("\\")[-1].lower()
-
-    if address_name:
-        address_name = " " + address_name
-    elif module:
-        address_name = " " + module
+    address_name = ""
+    module = dp.modules.find_module_by_address(address)

```

Comment:
Pretty sure this functionality changed. It’s only supposed to log the module when it changes.

Reply:
It was a mess of a function and at first was not sure what it was trying to do. I understand it now and made changes to `find_function_name` to make the trace dumps a lot nicer.

https://gist.github.com/oopsmishap/464dd43139c0876fd9f3ea1fba86cfd3

Reply:
The idea was that it will only print `handletest_x86.exe` when it switches there from a memory region not in that module. This way the log isn't overly spammy. Also the exports should be shown regardless of what the previous module was.

Reply:
Which one do you prefer?

With function offsets:
https://gist.githubusercontent.com/oopsmishap/7f56298e69b6dc6eaebea4d699b8860b/raw/3ce89017a9b9563a30ab00032c67d86f4e82acb1/Dit2QYnSgX.txt

Without function offsets:
https://gist.githubusercontent.com/oopsmishap/0ef74bf39fc4af9e4ef8531fd07220a6/raw/9027c1875a994fee6cc7058ca160683a57c7ca9c/fHwV7sdpxj.txt

Reply:
Without, in my opinion it should look exactly the same as before

Reply:
Ok, that's fine, the current state of the PR is without the function offsets.